### ISSUE TITLE
Use of spatial dropout

### ISSUE THREAD

**Issue by jlmaccal** [locuslab/TCN#54]
Could you clarify if spatial dropout is used? The paper suggests that it is, but the code seems to use standard dropout.

**Comment by jerrybai1995** [comment]
Hi @jlmaccal,

We originally did use the spatial dropout (which is simply `nn.Dropout1d` in pytorch), but then later it was pointed out to us that the standard dropout works better on language modeling tasks. Therefore the repo has been changed to use the standard dropout.

**Comment by jlmaccal** [comment]
Thanks for the clarification.
